### ISSUE TITLE
Events editor get wrong when opened and day change.

### ISSUE THREAD

**Issue by pasnox** [KDAB/Charm#259]
If u let open the "Events Editor" dialog during your work and a day pass to the next, then the dialog is not reliable anymore.
By example, the "Today" entry will still refer to yesterday and not the real Today date.
We need to close and reopen the dialog to make it works correctly.

**Comment by TheOneRing** [comment]
I tried to reproduce the behaviour.
For me the event is listed in yesterday and today, but the duration is still counted for the previous day.

**Comment by frankosterfeld** [comment]
That's correct, we display the event for both days currently (originally only the first day, changed to show it for both not too long ago). One could imagine to split the event at day boundaries when doing the daily sums for display (as long nothing is counted twice for the weekly sum etc...).

I think @pasnox sees an issue though with the event editor's filter not moving to the next day correctly when it stays open over midnight?

**Comment by TheOneRing** [comment]
The event editor was updated a few seconds after midnight.

**Comment by frankosterfeld** [comment]
@pasnox Which release or commit are you using?

**Comment by frankosterfeld** [comment]
The code checks the date every 60 seconds, so 00:01 latest the view should update. Confirmed to work here on OS X, timezone CET, current master.

**Comment by pasnox** [comment]
Let close this issue, can not reproduce anymore.
